src/connection.py

```diff
@@ -112,13 +112,26 @@ def on_message(self, payload):
                 self.timer.cancel()
             except:
                 pass
+        elif len(payload) == 2:
+            self.connection.write_message(chr(1), True)
         else:
             decompressed = zlib.decompress(payload)
             size = struct.unpack('=I', decompressed[:4])
             frmt = "%ds" % size[0]
             unpacked = struct.unpack('=I' + frmt, decompressed)
             data = json.loads(unpacked[1])
             self.values[data["Topic"]] = data
+            self.connection.write_message(chr(1), True)
+
+    def on_close(self):
+        self.connection = None
+        print "Server connection closed. Reconnecting..."
+        tornado.websocket.websocket_connect(
+                self.url,
+                self.ioloop,
+                callback = self.on_connected,
```

Comment:
This line and the line below does not follow PEP 8. No spaces allowed for assigning kwargs with `=` in a function call.